arangod/IResearch/IResearchLink.cpp

```diff
@@ -432,8 +432,8 @@ Result IResearchLink::init(velocypack::Slice definition,
                     std::to_string(_id.id()) + "'"};
       }
 
-      viewId = view->guid();  // ensue that this is a GUID (required by
-                              // operator==(IResearchView))
+      // ensure that this is a GUID (required by operator==(IResearchView))
+      viewId = view->guid();
```

Comment:
ASSERT ?

Reply:
No, see this PR https://github.com/arangodb/arangodb/pull/15658